src/lib/flex/layout-gap/layout-gap.ts

```diff
@@ -114,14 +113,12 @@ export class LayoutGapDirective extends BaseDirective2 implements AfterContentIn
     return buffer;
   }
 
-  constructor(protected elRef: ElementRef,
+  constructor(elRef: ElementRef,
               protected zone: NgZone,
               protected directionality: Directionality,
               protected styleUtils: StyleUtils,
```

Comment:
It seems like `protected` was removed from most of the other instances of `styleUtils`, but not this one. Can you share why? I'm just trying to better understand these changes.